client/device/duoadminpanel/delete.go

```diff
@@ -0,0 +1,20 @@
+package duoadminpanel
+
+import (
+	"context"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+)
+
+type DeleteInput struct {
+	Uid string
+}
+
+type DeleteOutput = device.DeleteOutput
+
+func Delete(ctx context.Context, client http.Client, deleteInp DeleteInput) (*DeleteOutput, error) {
+
+	client.Logger.Println("deleting duo admin panel")
+
+	return device.Delete(ctx, client, *device.NewDeleteInput(deleteInp.Uid))
```

Comment:
Ideally I would use the public API to do the delete too

Reply:
Yes agree, but I dont want to write something that is going to be deleted soon because we will have generated public api client for go-lang soon 🙂